tchannel/testing/dispatch.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import absolute_import
+
+from tornado import gen
+
+from ..response import response_from_mixed
+
+
+@gen.coroutine
+def handle(tchannel, request):
+    """Test util function to have tchannel instance to take predefined request
+    object.
+
+    .. code-block:: python
```

Comment:
The example is good but is missing some info. Like "assuming some tchannel handler foo, you can test it like so" and then go into detail like:

``
from ... import handle

def test_foo():
    ...
```